include/open62541/server_pubsub.h

```diff
@@ -629,6 +634,13 @@ typedef struct {
                        const UA_NodeId *targetVariableIdentifier,
                        void *targetVariableContext,
                        UA_DataValue **externalDataValue);
+    void (*write)(UA_Server *server,
+                       const UA_NodeId *readerIdentifier,
+                       const UA_NodeId *readerGroupIdentifier,
+                       const UA_NodeId *targetVariableIdentifier,
+                       void *targetVariableContext,
+                       void *data,
```

Comment:
The data and dataSize arguments are strange.
This should be wrapped into a Variant.

Reply:
Thanks, I have used UA_DataValue* now, hope this is ok as well